<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CONFileInfo</class>
 <widget class="QDialog" name="CONFileInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>393</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CON File Info</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artistLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>70</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Extract MIDI</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
